build-vm

```diff
@@ -915,6 +934,9 @@ vm_first_stage() {
 	    rmdir "$KERNEL_TEMP_DIR"
 	fi
     fi
+    if test "$USE_VIRT_FS" ; then 
+        /usr/bin/virt-make-fs --format=raw -t ext3 --size="$VMDISK_ROOTSIZE"M $BUILD_ROOT $VM_ROOT
```

Comment:
Just a random thought: shouldn't we clean $BUILD_ROOT after the virt-make-fs call?

Reply:
We could but we dont have to because there is `mkdir_build_root` call after `vm_setup` which should effectively cleanup `$BUILD_ROOT` before every build, which is the reason I left it.